terraform_providers/netconf/helper.go

```diff
@@ -0,0 +1,273 @@
+package netconf
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"sync"
+
+	driver "github.com/davedotdev/go-netconf/drivers/driver"
```

Comment:
Want you to consider using https://github.com/Juniper/go-netconf/. Send any relevant pull request from this private repo to that as needed.

Reply:
The point of this pr is to move away from using the Junos helper overlay from the go-netconfe library.

While it's still using that Library in the backend we are no longer reliant on the `helper` function.

Reply:
@davedotdev